BeVolt/Drivers/inc/LTC6811.h

```diff
@@ -357,8 +359,9 @@ int16_t LTC6811_run_adc_redundancy_st(uint8_t adc_mode,
                                       cell_asic ic[]);
 
 /*! Helper function that runs the datasheet open wire algorithm*/
-void LTC6811_run_openwire(uint8_t total_ic,
-                          cell_asic ic[]);
+uint32_t LTC6811_run_openwire_multi(uint8_t total_ic,
+                          cell_asic ic[],
+													bool print);
```

Comment:
This is a low-level driver. I don't think that this should depend on the UART driver in order to perform its function. I think that the better option here would be to have this driver return some information that a higher-level function could parse and output.

This might be alright, but just seems kind of odd to me. @SijWoo what do you think?

Reply:
It can be removed, but it was just easier to be able to test it efficiently